### PR TITLE
Added right image layout

### PR DESCRIPTION
**Pull request by prakash-zensar**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-            <li class="route route__navigation">
-              <router-link to="/header">Header</router-link>
-            </li>
-          </ul>
-        </div>
-        <div class="col-10 main__Container">
-          <router-view/>
-        </div>
+    <div class="row">
+      <div class="col-2 main__Route">
```
Comment (sadhna23) [inline src/App.vue:5]: @prakash-zensar use small case for class names i.e. main_route

File: src/router.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import Vue from "vue";
 import Router from "vue-router";
 import Home from "./views/Home.vue";
-import Layout from "./views/Layout.vue";
-import Header from "./views/Header.vue";
+// import Layout from "./views/Layout.vue";
```
Comment (sadhna23) [inline src/router.js:4]: @prakash-zensar remove comments

Comment (prakash-zensar) [reply src/router.js:4]: We need that commented code for future. We need this for run time binding of vew files

File: src/App.vue (hunk 1 above)
Comment (prakash-zensar) [reply src/App.vue:5]: Ashish is going to replace this file with his code as it is. I have added these for temporary as ashish is working on rouitng

**Pull request merged**